.travis.yml

```diff
@@ -1,16 +1,47 @@
 language: python
-python: 2.7
+python: 
+  - 2.7
+  - 3.5
 addons:
   firefox: 47.0.1
-env:
-- TOXENV=tests DISPLAY=:99.0
-- TOXENV=flake8
+matrix:
+  include:
+    - python: 2.7
+      env: 
+        - TOXENV=flake8
+    - python: 2.7
+      env: 
+        - DISPLAY=:99.0
```

Comment:
I got maybe mislead by the fact it ran test without environment variable. Now I exclude them, I should try again.